symphony-bdk-http/symphony-bdk-http-jersey2/src/main/java/com/symphony/bdk/http/jersey2/ApiClientBuilderJersey2.java

```diff
@@ -248,9 +262,32 @@ protected SSLContext createSSLContext() {
           .keyStorePassword(keyStorePassword);
     }
     try {
-      return sslConfig.createSSLContext();
+      SSLContext sslContext = sslConfig.createSSLContext();
+
+      // if logging debug is enabled, we print the truststore entries
+      if (logger.isDebugEnabled()) {
+        this.logTrustStore();
+      }
+
+      return sslContext;
     } catch (IllegalStateException e) {
-        throw new IllegalStateException(e.getCause().getMessage(), e);
+      throw new IllegalStateException(e.getCause().getMessage(), e);
+    }
+  }
+
+  private void logTrustStore() {
+    try {
+      if (isNotEmpty(trustStoreBytes) && isNotEmpty(trustStorePassword)) {
+        final KeyStore truststore = KeyStore.getInstance(TRUSTSTORE_FORMAT);
+        truststore.load(new ByteArrayInputStream(trustStoreBytes), trustStorePassword.toCharArray());
+        final List<String> aliases = Collections.list(truststore.aliases());
```

Comment:
can't we factor this code between the 2 implementation of the http client?

Reply:
Will be done in next commit